### PR TITLE
Add animations, theme and game over page

### PR DESCRIPTION
**Pull request by aruppert**
Refactored and deleted categories

### CODE REVIEW

File: client/src/components/Animations.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  80% {
+    transform: scale3d(1.1, 1.1, 1.1) rotate3d(0, 0, 1, -3deg);
+  }
+
+  to {
+    transform: scale3d(1, 1, 1);
+  }
+}
+
+.tada {
```
Comment (panchepanevski) [inline client/src/components/Animations.js]: Nice name :)

File: client/src/components/GameOver.js [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+const TextWrapper = styled.div`
+  position: absolute;
+  z-index: 1000;
+  display: flex;
+  top: 47vh;
+  left: 0px;
+  width: 100%;
+  align-content: center;
+  justify-content: center;
+  /* transform: translate(-50%, -50%); */
```
Comment (panchepanevski) [inline client/src/components/GameOver.js]: Remove comments ;)

File: client/src/components/GameOver.js [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+const ScoreCircle = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background: ${props => props.theme.colors.active};
+  border-radius: 50%;
+  height: 100px;
+  width: 100px;
+`;
+const ScoreTextWrapper = styled.p``;
```
Comment (panchepanevski) [inline client/src/components/GameOver.js]: Avoid empty components.

File: client/src/components/GameOver.js [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+
+const GameOverContainer = styled.div``;
```
Comment (panchepanevski) [inline client/src/components/GameOver.js]: Avoid empty components.

File: client/src/components/Input.js [hunk 5]
```diff
@@ -1,21 +1,14 @@
-import React from 'react';
 import styled from '@emotion/styled';
 
-const InputField = styled.input`
+const Input = styled.input`
+  background: ${props => props.theme.colors.background};
   border: none;
   border-radius: 5px 0px 5px 0px;
```
Comment (panchepanevski) [inline client/src/components/Input.js]: You don't need `border-radius` when the `border: none` is.

File: client/src/components/PublicConfirmationCard.js [hunk 6]
```diff
@@ -32,37 +33,52 @@ const NavButton = styled.button`
   font-weight: bold;
   margin: 0px 5px 0px 5px;
```
Comment (panchepanevski) [inline client/src/components/PublicConfirmationCard.js]: = as `margin: 0 5px` // 0 is for top and bottom, and 5px, right and left.

File: client/src/pages/MultiPlayerPage.js [hunk 7]
```diff
@@ -23,43 +23,77 @@ const AnswerContainer = styled.div`
...
+  width: 200px;
+  margin: 0px;
+  margin: 0 0 10px;
+  color: ${props => props.theme.colors.card1};
+`;
+const TextWrapperOutsideCard2 = styled.div`
+  text-align: center;
+  height: 30px;
+  width: 200px;
+  margin: 0px;
```
Comment (panchepanevski) [inline client/src/pages/MultiPlayerPage.js]: 2 x `margin`

File: client/src/components/GameOver.js (hunk 2 above)
Comment (aruppert) [reply client/src/components/GameOver.js]: Thx :D

File: client/src/components/GameOver.js (hunk 3 above)
Comment (aruppert) [reply client/src/components/GameOver.js]: thx!

File: client/src/components/GameOver.js (hunk 4 above)
Comment (aruppert) [reply client/src/components/GameOver.js]: thx!

File: client/src/components/Input.js (hunk 5 above)
Comment (aruppert) [reply client/src/components/Input.js]: ;) to shape I do.

File: client/src/components/PublicConfirmationCard.js (hunk 6 above)
Comment (aruppert) [reply client/src/components/PublicConfirmationCard.js]: Thx :D

File: client/src/pages/MultiPlayerPage.js (hunk 7 above)
Comment (aruppert) [reply client/src/pages/MultiPlayerPage.js]: :*

File: client/src/components/GameOver.js (hunk 3 above)
Comment (aruppert) [reply client/src/components/GameOver.js]: will use them later ;)

File: client/src/components/GameOver.js (hunk 4 above)
Comment (aruppert) [reply client/src/components/GameOver.js]: will use them later ;)

**Pull request merged**